### PR TITLE
aligned chat message Images

### PR DESCRIPTION
**Pull request by KumudiniK**

### CODE REVIEW

File: src/containers/Chat/ChatMessages/ChatMessage/ChatMessageType/ChatMessageType.module.css [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
 
 .Image img {
   overflow: hidden;
-  max-width: 200px;
+  max-width: 250px;
   height: auto;
   object-fit: contain;
+  text-align: center;
+}
+.video {
```
Comment (kurund) [inline src/containers/Chat/ChatMessages/ChatMessage/ChatMessageType/ChatMessageType.module.css:16]: Please follow uppercase naming convention for classes.

**Review by kurund — CHANGES_REQUESTED** [review]
Also would be good to have positive test coverage.

File: src/containers/Chat/ChatMessages/ChatMessage/ChatMessageType/ChatMessageType.module.css (hunk 1 above)
Comment (KumudiniK) [reply src/containers/Chat/ChatMessages/ChatMessage/ChatMessageType/ChatMessageType.module.css]: yes. Updated these changes .

**Comment by kurund** [comment]
@KumudiniK 

There are few display issues. See the below screenshot.

screenshot-glific test_3000-2021 05 07-00_24_10

**Review by mdshamoon — APPROVED** [review]

**Pull request merged**